### PR TITLE
[stable/kapacitor] Use an existing secret for the InfluxDB auth

### PR DESCRIPTION
**Pull request by afausti**
#### What this PR does / why we need it:

Sometimes you need to check in the values.yaml in a repo
and you don't want to expose the secrets. With the existingSecret
option you can use an existing Kubernetes secret with the keys
`influxdb-user` and `infuxdb-password` to set the corresponding
environment variables in Kapacitor. The keys are the same as the
ones used in the `influxdb-auth` secret created by the InfluxDB
Helm chart.

#### Checklist
[Place an '[x]' (no spaces) in all applicable fields. Please remove unrelated fields.]
- [x] [DCO](https://github.com/helm/charts/blob/master/CONTRIBUTING.md#sign-your-work) signed
- [x] Chart Version bumped
- [x] Variables are documented in the README.md
- [x] Title of the PR starts with chart name (e.g. `[stable/mychartname]`)

### CODE REVIEW

File: stable/kapacitor/Chart.yaml [hunk 1]
```diff
@@ -1,6 +1,6 @@
 apiVersion: v1
 name: kapacitor
-version: 1.1.3
+version: 1.1.4
```
Comment (Aisuko) [inline stable/kapacitor/Chart.yaml]: Add the parameter `existingSecret ` as the backward compatibility feature. I think we should bump the middle version would make more sense here.

**Review by Aisuko — CHANGES_REQUESTED** [review]
There may need some small changes

**Comment by Aisuko** [comment]
@afausti Please join the Kubernets/charts channel on Slack, and copied the PR address into the channel, there need someone who is the @helm member can have the permission to merge the PR.

**Comment by Aisuko** [comment]
The channel address please see the Slack address on the README.md at the project homepage.

**Comment by afausti** [comment]
I’ve asked on Kubernetes/charts channel and InfluxDB/kapacitor channel for help to merge this PR.

**Comment by Aisuko** [comment]
Looks good. Thanks for your patient.

**Review by Aisuko — APPROVED** [review]
/approve

**Comment by naseemkullah** [comment]
bump, PRs are piling up for this chart @Aisuko do you need a hand? add me as an owner if you want

**Comment by Aisuko** [comment]
I will make a discussion with helm members with merging this PR.

**Comment by Aisuko** [comment]
@afausti It's so long time. Thanks for your patient. I believe we can merge this after resolving the conflict.

**Comment by afausti** [comment]
Thanks @Aisuko, I've bumped the minor version of the chart as discussed above. It is now 1.2.0.

**Review by Aisuko — APPROVED** [review]
/LGTM

**Comment by Aisuko** [comment]
Thanks

**Pull request merged**